robot-server/tests/integration/http_api/persistence/test_compatibility.py

```diff
@@ -78,6 +70,10 @@ async def test_protocols_analyses_and_runs_available_from_older_persistence_dir(
 
             assert len(all_run_ids) == snapshot.expected_run_count
 
+            # Ideally, we would also fetch full commands via
+            # `GET /runs/{run_id}/commands/{command_id}`.
+            # We skip it for performance. On my machine, it would take ~7 seconds.
+
```

Comment:
Oh it looks like this comment is already here, on lines 88-90. You can delete this one.

```suggestion
            # Ideally, we would also fetch full commands via
            # `GET /runs/{run_id}/commands/{command_id}`.
            # We skip it for performance. On my machine, it would take ~7 seconds.

```

Reply:
fixed.